batcher/batcher/batcher.go

```diff
@@ -195,7 +229,7 @@ func (b *BatcherImpl) createSignedBatch(
 	multisigSignature, multisigFeeSignature []byte, confirmedTxs []eth.ConfirmedTransaction,
 ) *eth.SignedBatch {
 	firstTxNonceID, lastTxNonceID := uint64(0), uint64(0)
-	if generatedBatchData.BatchType != uint8(Consolidation) {
+	if generatedBatchData.BatchType < uint8(Consolidation) {
```

Comment:
Please make this more readable: if needed it is better to use multiple conditions with or (`||`)